homeassistant/components/homematic/cover.py

```diff
@@ -48,11 +54,20 @@ def set_cover_position(self, **kwargs):
 
     @property
     def is_closed(self):
-        """Return if the cover is closed."""
-        if self.current_cover_position is not None:
+        """Return whether the cover is closed."""
+        if isinstance(self._hmdevice, IPGarage):
+            return self._hmdevice.is_closed()
```

Comment:
You can not do IO inside the property. You need to read it out form data cache

Reply:
Thanks for the tip. Now I'm even getting the updates (which was the missing piece in the code before)